### ISSUE TITLE
Save callback is called upon initialization

### ISSUE THREAD

**Issue by av01d** [Simonwep/pickr#69]
#### Do you want to request a **feature** or report a **bug**? 
**bug**

#### What is the current behavior?
When I init a pickr and define a `save` callback, that callback is called when the pickr is being initialized, without any user interaction.

#### If the current behavior is a bug, please provide the steps to reproduce or create a [JSFiddle]
```
const pickr = new Pickr({
   el: $pickr[0],
   default: '#000',
   components: {
	preview: true,
	opacity: true,
	hue: true,
	interaction: {
		hex: true,
		rgba: true,
		input: true,
		clear: true,
		save: true
	}
   }
});
pickr.on('save', function(hsva, instance) {
   console.log('pickr.save', hsva); // Gets called upon initialization
});
```

The fix is simple: Line 89-90 in pickr.js, change
```
this._initializingActive = false;
this.setColor(this.options.default);
this._emit('init');
```
to
```
this.setColor(this.options.default);
this._initializingActive = false;
this._emit('init');
```

#### What is the expected behavior?
The `save` handler should not be called when the pickr is being initialized.

#### Your environment:
```
Pickr-version (see Pickr.version): 0.4.0 
Browser-version: All
Operating-system: All
```

**Comment by Simonwep** [comment]
Nah, wasn't sure if this would't be helpful. Anyway, changed in 22aa5e64b210c8d7d284b37c7a6e03bdd8538594 :)

**Comment by av01d** [comment]
The save handler is still being called on init when I set `default` to `null`.
See this fiddle: https://jsfiddle.net/av01d/q3n7rfyo/
(this fiddle uses pickr 0.4.6)
I really would like to be able to initialise pickr without a color being set (`default: null`) and without the save handler being called.

**Comment by Simonwep** [comment]
It's kinda a gray zone, changed a lot during different (major) versions... On the one side It could be useful if you need to perform a specific tast on change - including the default color. Another point would be that this should be _aggressively_ threaded as default.

I will take it into consideration / ask for other opinions.

**Comment by av01d** [comment]
Personally, I think this is a bug. The `save` handler is not called when initialized with any other `default` value than `null`. Only when `default = null`, that handler is called. That seems strange to me...

**Comment by Simonwep** [comment]
Hmm, your right... sorry - should've used TDD to prevent such things... nevermind haha, I'll fix it later - including another bug and release a new version.

**Comment by Simonwep** [comment]
Fixed, _ugh again_ here: 7b6e99e3ac5dea22788a4407651a9caffd6e90de
Hopefully I will keep this in mind when I got again such glorious ideas 😄
